docs/learning/tutorials/develop-web-viewer.md

```diff
@@ -7,17 +7,17 @@
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
 - Open the `your-app-name` directory in VS Code.
 - Follow these steps to obtain a new OIDC client to use in your viewer application:
-  1. Go to https://developer.bentley.com
+  1. Go to <https://developer.bentley.com>
   2. Click the **Sign In** button and sign-in using your Bentley account credentials
       - If you have not already registered, click **Register now** and complete the registration process.
   3. Click on your user icon and navigate to the **My Apps** page
   4. Click the **Register New** button
   5. Give your application a Name
   6. Select the **Visualization** API
   7. Select application type **SPA** (Single Page Web Application)
-  10. Enter **Redirect URL** `https://localhost:3000/signin-callback`
-  11. Leave post logout redirect URIs empty.
-  12. Click the **Save** button
+  8. Enter **Redirect URL** `https://localhost:3000/signin-callback`

```

Comment:
I believe this should be http: https://github.com/imodeljs/itwin-viewer/blob/116eece5541280c717afb675c0ecc8bd8f9ebeea/packages/modules/cra-template-itwin-viewer/template/.env#L3

```suggestion
  8. Enter **Redirect URL** `http://localhost:3000/signin-callback`

```

Reply:
thanks, updated both of them.